charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -346,7 +459,7 @@ data:
         {{- if or .Values.receivers.hostmetrics.enabled (or .Values.receivers.kubeletstats.enabled .Values.receivers.prometheus.enabled) }}
         metrics:
           receivers:
-            {{- if and .Values.receivers.hostmetrics.enabled (include "newrelic.common.privileged" .) }}
+            {{- if and .Values.receivers.hostmetrics.enabled (include "newrelic.common.privileged" .) (not (include "newrelic.common.lowDataMode" .)) }}
```

Comment:
The `and` should be removed.

Reply:
dont forget to make this change :D @dbudziwojskiNR